helpers/logrotate

```diff
@@ -88,8 +88,12 @@ $logfile {
     $su_directive
 }
 EOF
-    mkdir --parents $(dirname "$logfile")                              # Create the log directory, if not exist
-    cat ${app}-logrotate | $customtee /etc/logrotate.d/$app >/dev/null # Append this config to the existing config file, or replace the whole config file (depending on $customtee)
+    # Create the log directory, if not exist
+    mkdir --parents $(dirname "$logfile")
+    chmod 644 $(dirname "$logfile")
```

Comment:
Shouldnt `755` be more appropriate for a directory (c.f the `x` permission)

I'm not sure to understand if permissions for others really are necessary ? (Maybe that's because logrotate runs with a specific user)